CovertInfiltration/Src/CovertInfiltration/Classes/XComGameState_MissionSiteInfiltration.uc

```diff
@@ -371,8 +372,9 @@ function UpdateGameBoard()
 	// Do this before showing the screen to support max infil rewards
 
 	BonusTemplate = GetNextOverInfiltrationBonus();
-
-	if (BonusTemplate != none && GetCurrentInfilInt() >= GetNextThreshold())
+	TargetMilestone = GetNextThreshold();
```

Comment:
Wouldn't `NextThreshold` name be better? `TargetMilestone` makes me think that it's an `X2InfiltrationBonusMilestoneTemplate`